dataminer/Functions/SiteManager/SiteManagerGettingStarted.md

```diff
@@ -0,0 +1,179 @@
+---
+uid: SiteManagerGettingStarted
+---
+
+# Getting started with Site Manager
+
+## Site Manager DxM installation
+
+The Site Manager DxM must run on the same machine as the DataMiner Agent from which you want to connect to remote data sources. It can be used with any DataMiner System that uses version 10.5.10/10.6.0 or higher. You can [deploy it from the Admin app](xref:Managing_cloud-connected_nodes#deploying-a-dxm-on-a-dms-node).
+
+The Site Manager DxM is **by default included in the DaaS image of DataMiner 10.5.10**. If your DaaS system is using an older DataMiner version, you will need to [upgrade DataMiner](xref:Upgrading_a_DataMiner_Agent) and [deploy the Site Manager DxM](xref:Managing_cloud-connected_nodes#deploying-a-dxm-on-a-dms-node).
+
+When the Site Manager DxM has been installed, in the Windows services overview, you should see two services:
+
+- *DataMiner SiteManager*: This is the Site Manager DxM that interacts with DataMiner and the zrok-agent process
+- *zrok-agent*: This is the zrok Agent service with which the Site Manager DxM communicates for creating the communication tunnels
+
+## On-premises setup
+
+To allow DataMiner to access an on-premises data source via the Site Manager, the following steps are needed:
+
+### [Windows](#tab/windows)
+
+1. On a machine in your on-premises network, run the [DataMiner SiteManager setup](https://github.com/SkylineCommunications/dataminer-sitemanager-setup/blob/main/Setup-DataMinerSiteManager.ps1) script by using the following command, where you replace the placeholders with the **account token** and a **site name**.
+
+   ```powershell
+   iex "& { $(iwr https://raw.githubusercontent.com/SkylineCommunications/dataminer-sitemanager-setup/main/Setup-DataMinerSiteManager.ps1) } -Command install -AccountToken '<AccountToken>' -SiteName '<SiteName>'"
+   ```
+
+   - You can find the **account token** in the Site Manager logging in DataMiner Cube, via *Apps* > *System Center* > *Logging* > *Site Manager (DxM)*. This log file should contain a line mentioning a token as follows: `Your account token is aWsTbeKpwARK. You can now get started configuring your site(s). Learn more at https://aka.dataminer.services/SiteManagerGettingStarted."`. Copy this token.
+
+   - The **site name** should be a concise description of the site from which you are exposing data sources. When a connection is configured during the creation or editing of an element in DataMiner Cube, this description will be shown in the *Site* dropdown.
+
+   > [!NOTE]
+   >
+   > - The minimum OS version required by the script is Windows 10 or Windows Server 2019 build 17134.
+   > - You must run the script as administrator.
+   > - The machine where you install this script must be able to access the data sources you want to expose. It does not have to be the machine where the data sources are running.
```

Comment:
The fact that the machine has to be able to access the data sources is mentioned for Windows, but not for Linux. Is this OK?